javaslang/src/test/java/javaslang/collection/HashMultimapTest.java

```diff
@@ -287,7 +287,7 @@ public void shouldCreateSortedMapFrom10Pairs() {
 
     @Test
     public void shouldNarrowMap() {
-        final HashMultimap<Integer, Number> int2doubleMap = this.<Integer, Number> emptyMap().put(1, 1.0d);
+        final HashMultimap<Integer, Number> int2doubleMap = (HashMultimap<Integer, Number>) this.<Integer, Number> emptyMap().put(1, 1.0d);
```

Comment:
No, we have to cast in three places only, but Idea shows 33 usages of `emptyMap()`.
PR updated